nifi-nar-bundles/nifi-email-bundle/nifi-email-processors/src/main/java/org/apache/nifi/processors/email/ConsumeEWS.java

```diff
@@ -175,6 +175,16 @@
             .addValidator(StandardValidators.BOOLEAN_VALIDATOR)
             .build();
 
+    public static final PropertyDescriptor INCLUDE_EMAIL_HEADERS = new PropertyDescriptor.Builder()
+            .name("ews-include-headers")
+            .displayName("Original Headers to Include")
+            .description("Comma delimited list specifying which headers from the original message to include in the exported email message. '*' means copy all headers. " +
+                    "Some headers can cause problems with message parsing, specifically the 'Content-Type' header.")
```

Comment:
If excluding 'Content-Type' is the most use-case, user may want to simply excludes it. It can be better if it provides both of 'include' and 'exclude' headers options. There are many headers possibly returned, https://www.iana.org/assignments/message-headers/message-headers.xhtml